app/models/katello/glue/candlepin/owner.rb

```diff
@@ -21,7 +21,9 @@ def owner_info
       end
 
       def owner_details
-        Resources::Candlepin::Owner.find self.label
+        details = Resources::Candlepin::Owner.find self.label
+        details['virt_who'] = self.subscriptions.any?(&:virt_who?)
```

Comment:
I'm still a little concerned (maybe less so than below) about the performance of this, depending on the # of subscriptions it could be slow, you could rewrite it like:

```
self.subscriptions.joins(:pools).where("#{Katello::Pool.table_name}.virt_who" => true).any?
```